### DISCUSSION TITLE
Code blocks for Wysiwyg

### DISCUSSION THREAD

**Discussion by iksent** [directus/directus#5963] [Feature Requests]
Hello, team.
What about adding a codesample plugin for Directus Wysiwygs?
https://www.tiny.cloud/docs/plugins/opensource/codesample/

**Comment by azrikahar** [comment]
I believe the answer from https://github.com/directus/directus/discussions/12327#discussioncomment-2423262 is technically applicable here.

---

Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
